### ISSUE TITLE
A11y Update: Change Badge component "Info" variant text color to black

### ISSUE THREAD

**Issue by gabew1984** [openedx/paragon#1428]
https://paragon-openedx.netlify.app/components/badge/

"Info" variant of the `Badge` component has white text that doesn't pass accessibility contrast standards. Update the text color of the Info variant to `$gray-900`

**Comment by adamstankiewicz** [comment]
@gabew1984 Presumably we only want to make this color change for [the edX.org theme](https://github.com/edx/brand-edx.org), not the core/default Open edX theme (i.e., it should remain white in the Open edX theme), correct?

<img width="563" alt="image" src="https://user-images.githubusercontent.com/2828721/176084419-3e32bb9a-4236-4cfc-9009-13fa11c8029e.png">

**Comment by adamstankiewicz** [comment]
This is confirmed ⬆️

**Comment by adamstankiewicz** [comment]
Completed via https://github.com/edx/brand-edx.org/pull/44
